pages/index.html

```diff
@@ -1,12 +1,41 @@
 <!DOCTYPE html>
```

Comment:
It is a convention to put the `index.html` at the project root directory level and additional pages in a directory such as `pages`